test/test_testing.py

```diff
@@ -2699,6 +2700,99 @@ def test_container_pebble_ready(self):
         )
 
 
+class TestNetwork(unittest.TestCase):
+    def test_add_network_defaults(self):
+        harness = Harness(CharmBase)
```

Comment:
I'm not that familiar with the unittest framework, could this go into `setUp` or something like that? It looks like something that would be needed on most tests, similar for `addCleanup`